src/app/components/nav/simple-header/simple-header.component.ts

```diff
@@ -0,0 +1,149 @@
+import {ChangeDetectorRef, Component, ElementRef, HostListener, Inject, OnDestroy, OnInit, ViewChild} from '@angular/core';
+import {GlobalService} from '../../../services/global.service';
+import {ActivatedRoute, Router} from '@angular/router';
+import {AuthService} from '../../../services/auth.service';
+import {ApiService} from '../../../services/api.service';
+import {DOCUMENT} from '@angular/common';
+
+@Component({
+  selector: 'app-simple-header',
+  templateUrl: './simple-header.component.html',
+  styleUrls: ['./simple-header.component.scss']
+})
+export class SimpleHeaderComponent implements OnInit, OnDestroy {
+
+  user = {username: ''};
```

Comment:
Just want to let you know - If you want to create an object then it would be better to user Angular interface concept and make a model in a separate file.

Reply:
Okay, I will keep that in mind.